app/V1Module/security/Policies/ReferenceExerciseSolutionPermissionPolicy.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace App\Security\Policies;
+
+
+use App\Model\Entity\ReferenceExerciseSolution;
+use App\Security\Identity;
+
+class ReferenceExerciseSolutionPermissionPolicy implements IPermissionPolicy {
+  public function getAssociatedClass() {
+    return ReferenceExerciseSolution::class;
+  }
+
+  public function isAuthor(Identity $identity, ReferenceExerciseSolution $referenceExerciseSolution = null) {
+    if ($referenceExerciseSolution === null) {
```

Comment:
We use `NULL`, not `null`. The same goes for `true` and `false`.

Reply:
That is in violation of http://www.php-fig.org/psr/psr-2/

Reply:
null, true and false MUST be in lowercase

Reply:
Well, if you want to search for violations of PSR, there are plenty more :) IMO it's more important to be consistent... But this is a minor nitpick and it's possible we don't follow that rule so strictly anyway.

Reply:
Consistency is broken anyways. Lets shift this issue into another task (which will ensure consistency in one way or another).

Reply:
Agreed, have my approval :)